test/views/email/CaptureEmailViewSpec.scala

```diff
@@ -71,10 +71,6 @@ class CaptureEmailViewSpec extends ViewBaseSpec {
           elementText(Selectors.pageHeading) shouldBe "What is the email address?"
         }
 
-        "have the correct hint text" in {
```

Comment:
I can see why you think we might not need this because we would essentially be testing the helper that is provided for us. However, I think there is value in keeping it because it ensures that we are displaying the correct content for the hint. Happy to debate this.